_bookdown.yml

```diff
@@ -1,13 +1,13 @@
-book_filename: "ITCR_Course_Name"
+book_filename: "ITCR_Documentation_and_Usability"
 chapter_name: "Chapter "
-repo: https://github.com/jhudsl/ITCR_Course_Template/
-rmd_files: ["index.Rmd", 
-            "01-intro.Rmd", 
-            "02-chapter_of_course.Rmd", 
+repo: https://github.com/jhudsl/ITCR_Documentation_and_Usability/
```

Comment:
Do you think we can add this to the YAML automatically?

Reply:
Oh. Maybe. What would you suggest?

Reply:
Maybe:
```r
update_repo = function(file = "_bookdown.yml") {
y = yaml::yaml.load_file(file)
y$repo = gert::git_remote_info()$url
yaml::write_yaml(y, file)
}
```

Reply:
Where would this go?

Reply:
I think it'd be a function we're wrapping like the ones in the other repo.

Reply:
I'm going to file a separate issue about this in the bookdown repo. I think if we can get a collection of these items, it would be helpful for filling in items upon using the template. But I'm hesitant to add this code for just one line that's pretty easily changed.

Reply:
https://github.com/jhudsl/ITCR_Course_Template_Bookdown/issues/39